bot/seasons/easter/__init__.py

```diff
@@ -3,14 +3,23 @@
 
 class Easter(SeasonBase):
     """
-    Easter is a beautiful time of the year often celebrated after the first Full Moon of the new spring season.
-    This time is quite beautiful due to the colorful flowers coming out to greet us. So. let's greet Spring
-    in an Easter celebration of contributions.
+    Easter is a beautiful time of the year often celebrated after the first full moon of the
+    Northern Hemisphere's new spring season, making it a beautiful time of the year with colorful
```

Comment:
Easter is a beautiful time of the year [...], making it a beautiful time of the year.